### PR TITLE
Graphical Informations update

### PR DESCRIPTION
**Pull request by mpoudret**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- Add wellbore completion md uom to string getter
- Add missing items to swig files
- Fix hdf permission access in EtpDocument 
- Fix content type test issues in EpcDocument
- Since 08/23/2018 Resqml 2.2 schema GraphicalInformationSet.ValueVectorIndex attribute is optional: Hanlding of this attribute have be updated according to this change and Eml 2.2 proxies have been regenerated according to the 08/23/2018 Resqml 2.2 schema

Does this close any currently open issues?
------------------------------------------
No

Any relevant logs, error output, etc?
-------------------------------------
No

Any other comments?
-------------------
NO

Where has this been tested?
---------------------------
**Operating System:** Windows 10 64bits

**Platform:** Visual Studio Community 2019

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -323,4 +323,5 @@ __pycache__/
 *.xsd.cs
 
 # OpenCover UI analysis results
-OpenCover/
\ No newline at end of file
+OpenCover/
+swig/swigModule.i
```
Comment (philippeVerney) [inline .gitignore]: already present line 4

File: src/common/GraphicalInformationSet.cpp [hunk 2]
```diff
@@ -497,7 +497,11 @@ void GraphicalInformationSet::setDiscreteColorMap(AbstractObject* targetObject,
 
 	colorInformation->UseReverseMapping = useReverseMapping;
 	colorInformation->UseLogarithmicMapping = useLogarithmicMapping;
-	colorInformation->ValueVectorIndex = valueVectorIndex;
+
+	if (colorInformation->ValueVectorIndex == nullptr) {
+		colorInformation->ValueVectorIndex = soap_new_LONG64(gsoapProxy2_2->soap, 1);
```
Comment (philippeVerney) [inline src/common/GraphicalInformationSet.cpp]: I think the second parameter of soap_new... is not needed (even if we use it often)

File: src/common/GraphicalInformationSet.cpp [hunk 3]
```diff
@@ -595,7 +599,11 @@ void GraphicalInformationSet::setContinuousColorMap(AbstractObject* targetObject
 
 	colorInformation->UseReverseMapping = useReverseMapping;
 	colorInformation->UseLogarithmicMapping = useLogarithmicMapping;
-	colorInformation->ValueVectorIndex = valueVectorIndex;
+	
+	if (colorInformation->ValueVectorIndex == nullptr) {
+		colorInformation->ValueVectorIndex = soap_new_LONG64(gsoapProxy2_2->soap, 1);
```
Comment (philippeVerney) [inline src/common/GraphicalInformationSet.cpp]: same

File: swig/swigResqml2_0_1Include.i [hunk 4]
```diff
@@ -140,7 +140,6 @@ namespace std {
    %template(StratigraphicUnitInterpretationVector) vector<RESQML2_0_1_NS::StratigraphicUnitInterpretation*>;
    %template(StratigraphicOccurrenceInterpretationVector) vector<RESQML2_0_1_NS::StratigraphicOccurrenceInterpretation*>;
    %template(HorizonInterpretationVector) vector<RESQML2_0_1_NS::HorizonInterpretation*>;
-   %template(ConstGrid2dRepresentationVector) vector<RESQML2_0_1_NS::Grid2dRepresentation const*>;
```
Comment (philippeVerney) [inline swig/swigResqml2_0_1Include.i]: Should it not be replaced by the non const version?

File: .gitignore (hunk 1 above)
Comment (mpoudret) [reply .gitignore]: Done

File: src/common/GraphicalInformationSet.cpp (hunk 2 above)
Comment (mpoudret) [reply src/common/GraphicalInformationSet.cpp]: Done

File: src/common/GraphicalInformationSet.cpp (hunk 3 above)
Comment (mpoudret) [reply src/common/GraphicalInformationSet.cpp]: Done

File: swig/swigResqml2_0_1Include.i (hunk 4 above)
Comment (mpoudret) [reply swig/swigResqml2_0_1Include.i]: Sure. Done.

File: .gitignore (hunk 1 above)
Comment (mpoudret) [reply .gitignore]: Done.

File: src/common/GraphicalInformationSet.cpp (hunk 2 above)
Comment (mpoudret) [reply src/common/GraphicalInformationSet.cpp]: Done.

File: src/common/GraphicalInformationSet.cpp (hunk 3 above)
Comment (mpoudret) [reply src/common/GraphicalInformationSet.cpp]: Done.

File: swig/swigResqml2_0_1Include.i (hunk 4 above)
Comment (mpoudret) [reply swig/swigResqml2_0_1Include.i]: Sure. Done.

**Pull request merged**